src/main/webapp/i18n/en/global.json

```
@@ -170,9 +170,12 @@
         "REFERENCE": "Please upload your References",
         "CV": "Please upload your CV"
       },
+      "add": "Add",
+      "clear": "Clear",
+      "upload": "Upload",
       "uploaded_document": "Uploaded Document for ",
       "uploaded_documents": "Uploaded Documents for ",
-      "instruction": "Click or drag and drop files to here to upload.",
+      "instruction": "Click Add or drag and drop files to here to upload.",
```

Please use single quotation marks, similarly to how you did it for the german translation
```suggestion
      "instruction": "Click on 'Add' or drag and drop files to here to upload.",
```

---

The Add button was removed, so the text changed - this is now not necessary anymore